src/platform/3ds/3ds_ui.cpp

```diff
@@ -126,9 +123,9 @@ CtrUi::CtrUi(int width, int height, const Game_ConfigVideo& cfg) : BaseUi(cfg)
 #endif
 
 #ifdef NDEBUG
-	// Loading bottom screen keyboard
-	assets = C2D_SpriteSheetLoadFromMem(assets_t3x, assets_t3x_size);
-	bottom_image = C2D_SpriteSheetGetImage(assets, assets_keyboard_idx);
+	// FIXME: Loading bottom screen keyboard, no spritesheet!
```

Comment:
This was actually a hack to begin with. Using a spritesheet with only one image to avoid setting up all texture and image structs. `ctr_add_graphics_target` does not like this (and while we could work around it), doing it manually is likely better. Will come up with a solution for console vs. keyboard anyway, since cmake does not add `NDEBUG`, likely because of some custom flags.